src/server/create-graph.ts

```diff
@@ -455,7 +455,7 @@ for (const node of nodes) {
 }
 
 const commit = await $`git log -1 --format="%H %ct"`
-  .cwd(Bun.fileURLToPath(import.meta.resolve("../../../content")))
+  .cwd(Bun.fileURLToPath(import.meta.resolve(CONTENT_ROOT)))
```

Comment:
Good catch but we now have two: `BUILT_CONTENT_ROOT` and `CONTENT_SOURCE_ROOT`, and you would need to use `CONTENT_SOURCE_ROOT` here.

Reply:
```suggestion
  .cwd(Bun.fileURLToPath(import.meta.resolve(CONTENT_SOURCE_ROOT)))
```